bretm9/genrefy
# Nd typescript template

### What’s this PR do?  

- Changes the previous react-app installation to react-app with typescript
 
### Where should the reviewer start?  

- Git pull this branch.
 
### How should this be manually tested?  

- Git pull.
- `npm install` if first time getting repo
- `npm start` to ensure no error messages
 
### Any background context you want to provide?  

- When installing create-react-app with typescript, don't do `--template-typescript`, only `--template typescript` as the extra hyphen will not install typescript.
 
### What are the relevant tickets?  

- closes #6
 
### Screenshots (if appropriate)  

- N/A 

### Questions: 

- N/A
